config/development.ini

```diff
@@ -130,8 +135,8 @@ solr_url = http://solr:8983/solr/inventory
 # Note: Add ``datastore`` to enable the CKAN DataStore
 #               Add ``resource_proxy`` to enable resource proxying and get around the
 #               same origin policy
-# TODO: re-integrate datagov_inventory, dcat_usmetadata, usmetadata, datajson, saml2auth
-ckan.plugins = datastore xloader stats text_view recline_view googleanalyticsbasic s3filestore envvars
+# TODO: re-integrate dcat_usmetadata, usmetadata, datajson, saml2auth
+ckan.plugins = datagov_inventory datastore xloader stats text_view recline_view googleanalyticsbasic s3filestore envvars
```

Comment:
This is all good, but it's not added to production.ini

Reply:
👍